libs/core/src/lib/token/tokenizer.component.html

```diff
@@ -3,7 +3,7 @@
     [class.is-focus]="_tokenizerHasFocus"
     [class.fd-tokenizer--compact]="compact"
 >
-    <div class="fd-tokenizer__inner" #tokenizerInner>
+    <div class="fd-tokenizer__inner" role="listbox" #tokenizerInner>
```

Comment:
It probably needs aria-labelled by

Reply:
@InnaAtanasova , it's intended to read item count for of tokens available. You suggest using aria-labelled for this right?